sms_939/models/sms_notification.py

```diff
@@ -123,9 +123,11 @@ def run_service(self):
                     })
         if 'test' not in sms_text:
             self.env['sms.sender.wizard'].create({
-                'text': sms_answer
-                # large account id is 1, small account id is 2
-            }).send_sms(mobile=sms_receipient, provider_account_id=2)
+                'text': sms_answer,
+                'sms_provider': self.env['sms.provider'].search([
+                    ('name', 'ilike', 'small account')
```

Comment:
Not a good idea either, because we have to make sur that we name our provider  "small account", or this will fail to find the good one. I think the best way to differentiate the two providers is to add a xml data file and create the two records (and fill the fields "name", "server", "port", "endpoint" but put dummy values in fields "username" and "password").
Then here and other places where you need to reference the correct provider, you can use the function `self.env.ref('sms_939.provider_xml_id')` fo fetch the correct provider.